includes/derivatives.inc

```diff
@@ -74,6 +75,18 @@ function islandora_large_image_create_jp2_derivative(AbstractObject $object, $fo
         }
       }
 
+      if (variable_get('islandora_convert_rgb8', FALSE)) {
+        // Make a copy of the source that can change
+        $uploaded_file = str_replace("temporary://", file_directory_temp() . '/', $uploaded_file);
+        $normalized_src = file_create_filename(drupal_basename($uploaded_file) . ".tif", file_directory_temp());
+        $normalized_src = str_replace("temporary://", file_directory_temp() . '/', $normalized_src);
+        $convert_command = 'convert ' . $uploaded_file . ' -compress None -resample 300x300 -depth 8 ' . $normalized_src;
```

Comment:
Why is there a 300 by 300 resample there?

Reply:
@bgilling